src/components/common/Button/index.tsx

```diff
@@ -1,37 +1,46 @@
-import { FC, ReactNode, MouseEvent } from 'react';
+import { FC, MouseEvent, ReactNode } from 'react';
 
 export interface ButtonProps {
+  // eslint-disable-next-line no-unused-vars
   onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
   children: ReactNode;
   color: string;
   size?: string;
 }
 
+// eslint-disable-next-line no-unused-vars
 enum Colors {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
If you set eslint settings, why do you still need this line?

Reply:
it is just for interface and enum
if you know how to set no-unused-vars for interface and enum, assist me